components/scream/tests/uncoupled/CMakeLists.txt

```diff
@@ -7,13 +7,11 @@ if (NOT ${SCREAM_BASELINES_ONLY})
   add_subdirectory(p3)
   add_subdirectory(shoc)
   add_subdirectory(cld_fraction)
-  if (DEFINED SCREAM_SPA_DATA_DIR)
-    add_subdirectory(spa)
-  endif()
+  add_subdirectory(spa)
   if (SCREAM_DOUBLE_PRECISION)
-      add_subdirectory(rrtmgp)
+    add_subdirectory(rrtmgp)
   else()
-      message(STATUS "WARNING: RRTMGP only supported for double precision builds; skipping")
+    message(STATUS "WARNING: RRTMGP only supported for double precision builds; skipping")
```

Comment:
While at it, can you remove the word "WARNING" from this? CTest catches it as a warning for the config phase, when reporting to cdash. Since there's nothing the user can do to avoid skipping rrtmgp for sp runs, we could simply notify the user:

```
message (STATUS: "RRTMGP only supported for double precision builds. Skipping")
```

Reply:
DOne.